.editorconfig

```diff
@@ -1,2 +1,3 @@
 [*.{kt,kts}]
-ktlint_code_style = android_studio
```

Comment:
> Furthermore the status bar (showing the time, notification icons, ...) normally has the same color than the app bar, only on some pages it is black. I couldn't figure out yet why that is the case.

The only page is the share/push messages view right? It seems to be related to the scrollview. Wrapping the scrollview in a androidx.drawerlayout.widget.DrawerLayout like the other pages, seems to fix this.